change/@itwin-grouping-mapping-widget-0f16e7f9-d417-4f4c-b408-e00876e39988.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "updated insights client to 0.10.1",
+  "packageName": "@itwin/grouping-mapping-widget",
+  "email": "141143406+julioalvahash@users.noreply.github.com",
+  "dependentChangeType": "patch"
```

Comment:
This should be a minor bump.

Reply:
Running `pnpm change` and selecting type `minor` doesn't change `dependentChangeType`  for me. It is supposed to change because of the dependency update in the grouping-and-mapping widget or because the `type` is a minor change?

Reply:
Just manually change it in the file to `minor`

Reply:
There is enough changes to justify this being a minor change, insights-client update is one of them.

Reply:
Changed to minor.

Reply:
Got it, I don't see any other itwin packages that depend on `@itwin/grouping-mapping-widget : workspace` in this repo other than the test viewer, is that why `dependentChangeType` is `patch` and if there were, would the `dependentChangeType` be controlled by `pnpm change` ?, I'm just curious about it. I'll revert this change.

Reply:
Oh my bad. I saw the last line first. Thinking it was the actual change type. Thanks @saskliutas